assets/model_monitoring/components/src/model_data_collector_preprocessor/run.py

```diff
@@ -87,55 +86,65 @@ def mdc_preprocessor(
     )
 
     # Read mltable from preprocessed_data
-    df = read_mltable_in_spark(mltable_path=des_path)
+    df = try_read_mltable_in_spark(des_path, "preprocessed_data")
 
-    if df.count() == 0:
-        raise Exception(
-            "The window for this current run contains no data. "
+    if not df:
+        print(" ''.")
```

Comment:
Message?

Reply:
yeap, I'll need to add a message, will defer to the next one to get this big PR in